lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1775,6 +1775,9 @@ def cmd_sessions(*args)
             elsif session.type == 'shell'
```

Comment:
This should probably be an `|| session.type == 'powershell'` as the code is identical.